src/traces/heatmap/calc.js

```diff
@@ -0,0 +1,432 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var isNumeric = require('fast-isnumeric');
+
+var Plotly = require('../../plotly');
+var Lib = require('../../lib');
```

Comment:
`Lib` is free of circular imports. :tada:  So, let's try import it directly from now on.